benchmark/HardwareIntrinsicsBenchmark/Tests.cs

```diff
@@ -9,10 +9,109 @@
 using System.Runtime.InteropServices;
 using BenchmarkDotNet.Attributes;
 
+[module: newmsgpack::MessagePack.UnsafeUnmanagedStruct(typeof(Benchmark.Matrix4x4), newmsgpack::MessagePack.UnsafeUnmanagedStructFormatterImplementationKind.Array | newmsgpack::MessagePack.UnsafeUnmanagedStructFormatterImplementationKind.Array)]
```

Comment:
Why are you bitwise OR'ing the `MessagePack.UnsafeUnmanagedStructFormatterImplementationKind.Array` flag twice?

Reply:
It is a mistake.

Reply:
The same mistake remains in this PR's description.